_posts/2018-10-08-Dask-Jobqueue.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Your Title
```

Comment:
You'll probably want to change this title

    title: Dask-Jobqueue
    tagline: Easily deploy on ...

Also, we recently added an author field.  You might want to add the following line here in the header material

    author: Joe Hamman